### PR TITLE
server: refine pprof sql in prepare stmt

### PR DESCRIPTION
**Pull request by lysu**
### What problem does this PR solve? 

prepare stmt is no need to re normalize and no need append args

### What is changed and how it works?

add a new method to generate pprof label

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - impl

Side effects

 - n/a

Related changes

 - Need to cherry-pick to the release branch

Release note

 - n/a

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/14425)

### CODE REVIEW

File: server/conn_stmt.go [hunk 1]
```diff
@@ -644,3 +644,20 @@ func (cc *clientConn) preparedStmt2String(stmtID uint32) string {
 	preparedAst := preparedObj.PreparedAst
 	return preparedAst.Stmt.Text() + sv.PreparedParams.String()
 }
+
+func (cc *clientConn) preparedStmt2StringNoArgs(stmtID uint32) string {
```
Comment (jackysp) [inline server/conn_stmt.go]: ... we could combine it with the above function.

Comment (lysu) [reply server/conn_stmt.go]: done, now let `preparedStmt2String` call `preparedStmt2StringNoArgs`

I didn't choose add a "flag param" to `preparedStmt2String`(which lead many lines be modified which make other reviewer unhappy) or add `opt func param` to `preparedStmt2String`(which is too heavy for this simple function)

**Comment by lysu** [comment]
@coocood @jackysp  PTAL

**Pull request merged**